astropy/time/core.py

```diff
@@ -464,6 +499,35 @@ def __repr__(self):
     def __str__(self):
         return str(getattr(self, self.format))
 
+    def strftime(self, format_spec):
+        """
+        Convert Time to a string according to a format specification.
+        See time.stftime documentation for format specification.
+
+        Parameters
+        ----------
+        format_spec : string
+            Format definition of return string.
+
+        Returns
+        -------
+        string
+            Formatted string according to the given format string.
+
+        """
+        formatted_strings = []
+        for sk in list(self.replicate('iso')._time.str_kwargs()):
```

Comment:
Is this use of `list` necessary for some reason?  I think it should work fine without it and save creation of a temporary list of the whole time object.